ehrapy/anndata/anndata_ext.py

```diff
@@ -579,6 +641,25 @@ def _update_uns(
         return all_moved_num_columns, list(all_moved_non_num_columns), None
 
 
+def _create_new_var(adata: AnnData, cols_not_in_x: list[str]) -> pd.DataFrame:
+    """Create a new var DataFrame with the EHRAPY_TYPE_KEY column set for entries from .obs
```

Comment:
```suggestion
    """Create a new var DataFrame with the EHRAPY_TYPE_KEY column set for entries from .obs.
```

Maybe I should enable some docstring linting :joy: